### ISSUE TITLE
I don't want to use  model = LogisticRegression(random_state=seed),but a model defined by myself.

### ISSUE THREAD

**Issue by mrlinyj** [Gogul09/flower-recognition#8]
how to Sequential to define a model  to insert the code?
Can you give me a example or some suggestion?

**Comment by mrlinyj** [comment]
why ？

**Comment by mrlinyj** [comment]
what means "weights"  used "imagenet",

**Comment by mrlinyj** [comment]
I want to konw the retrain code,How can I get it

**Comment by mrlinyj** [comment]
Can I using the other dataset?

**Comment by Gogul09** [comment]
Hi, can you please follow the `Issues and Workarounds` section in [this](https://gogul09.github.io/software/flower-recognition-deep-learning) tutorial post?

`custom` is a layer name that we assign by editing the model of our choice in `applications `module of Keras.

**Comment by mrlinyj** [comment]
model = Sequential()
model.add(Dense(6, activation='softmax',input_dim=4096))
model.compile(optimizer='rmsprop',
              loss='categorical_crossentropy',
              metrics=['accuracy'])
model.fit(trainData, trainLabels, epochs=10, batch_size=32)

VGG16
 Error when checking target: expected dense_3 to have shape (6,) but got array with shape (1,)

**Comment by Gogul09** [comment]
You can try with different classifiers from sklearn or Keras. Go [here](http://scikit-learn.org/stable/modules/classes.html) and find **classifier** and read the syntax for each classifier in sklearn. Some classifiers wont have `predict_probab`.

For this problem, **logistic regression** gave me good results. You can check [this](https://gogul09.github.io/software/image-classification-python) post on how to use different sklearn classifiers for this flower species recognition problem in python.

**Comment by Gogul09** [comment]
When using a Neural Network model using Keras, you need to give correct `input_dim` and specify correct **output classes** in the final `softmax` layer. Else you will get problems like that. Please refer to [this](https://gogul09.github.io/software/digits-recognition-mlp) example on how to use `Sequential() ` for **handwritten digits recognition problem** which could be applied here too.

**Comment by MitPatel971997** [comment]
can you please give code of model you trained ? 
email: mit9797@icloud.com
